x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/categorization_view/category_stopped_partitions.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, useContext, useEffect, useState, useMemo } from 'react';
+import { EuiBasicTable, EuiCallOut, EuiSpacer, EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+import { Subscription } from 'rxjs';
+import { JobCreatorContext } from '../../../job_creator_context';
+import { CategorizationJobCreator } from '../../../../../common/job_creator';
+import { Results } from '../../../../../common/results_loader';
+import { ml } from '../../../../../../../services/ml_api_service';
+
+const NUMBER_OF_PREVIEW = 5;
+export const CategoryStoppedPartitions: FC = () => {
+  const { jobCreator: jc, resultsLoader } = useContext(JobCreatorContext);
+  const jobCreator = jc as CategorizationJobCreator;
+  const [tableRow, setTableRow] = useState<Array<{ partitionName: string }>>([]);
+  const [hasStoppedPartitions, setHasStoppedPartitions] = useState(false);
+
+  let _resultsSubscription: undefined | Subscription;
+
+  const columns = useMemo(
+    () => [
+      {
+        field: 'partitionName',
+        name: i18n.translate(
+          'xpack.ml.newJob.wizard.pickFieldsStep.stoppedPartitionsPreviewColumnName',
+          {
+            defaultMessage: 'Stopped partition {count, plural, one {name} other {names}}',
+            values: { count: tableRow.length },
+          }
+        ),
+        render: (partition: any) => (
+          <EuiText size="s">
+            <code>{partition}</code>
+          </EuiText>
+        ),
+      },
+    ],
+    []
+  );
+  function setResultsWrapper(results: Results) {
+    loadCategoryStoppedPartitions();
+  }
+
+  async function loadCategoryStoppedPartitions() {
+    const results = await ml.results.getCategoryStoppedPartitions([jobCreator.jobId]);
```

Comment:
This fires when the job doesn't yet exist, which results in an uncaught error.
This error should be caught. if it's a 404 it can be ignored.

Reply:
Updated here https://github.com/elastic/kibana/commit/676dfff60738922e0ffdb7fa6c192fcc588c9a0d